pkg/sessions/redis/redis_store.go

```diff
@@ -111,8 +111,9 @@ func newRedisCmdable(opts options.RedisStoreOptions) (Client, error) {
 // Save takes a sessions.SessionState and stores the information from it
 // to redies, and adds a new ticket cookie on the HTTP response writer
```

Comment:
It'd be nice to get some tests written for this code, but I don't think it should block this change.